plugins/obs-ffmpeg/obs-ffmpeg-source.c

```diff
@@ -521,6 +523,12 @@ static void ffmpeg_source_update(void *data, obs_data_t *settings)
 	dump_source_info(s, input, input_format);
 	if (!s->restart_on_activate || active)
 		ffmpeg_source_start(s);
+
+	const bool force_play_start = obs_data_get_bool(settings, "force_play_start");
+	if (s->is_local_file && !active && input && force_play_start) {
+		ffmpeg_source_play_pause(s, false);
+		obs_data_set_bool(settings, "force_play_start", false);
```

Comment:
should we update "force_play_start" before calling ffmpeg_source_play_pause() as this function too can update it?

Reply:
Yes, sounds reasonable